recipes/multiprocessing-logging/meta.yaml

```diff
@@ -0,0 +1,39 @@
+{% set name = "multiprocessing-logging" %}
+{% set version = "0.3.1" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://github.com/jruere/multiprocessing-logging/archive/1b80bd458bec4bcf0f312c5fbdf1088fef7235fb.zip"
```

Comment:
Please use a URL with the version in it. This will enable our bots to do automatic maintenance.

Reply:
It seems this project does not have any release. Can you please generate a release for it?

Reply:
URL now has the version in it. I don't think a true Github Release was done, but the repo was tagged. Is that sufficient?